README.md

```diff
@@ -143,7 +144,7 @@ grunt watch
 npm link
 
 cd Foo
-npm install --save git://github.com/turn/Smart-Table.git#v0.3.8
+npm install --save git://github.com/turn/Smart-Table.git#v0.3.12
```

Comment:
this is minor. we need a little regex to get the major and minor version from package.json and put it in the template. Mabye we should do it later.